components/Styles.js

```diff
@@ -0,0 +1,43 @@
+import { StyleSheet } from 'react-native';
+
+export default {
+  styles: StyleSheet.create({
```

Comment:
I imagine we're only going to have the one stylesheet in here so lets export it directly rather than wrapping it in an Object. That way we can still use eg styles.show instead of Styles.styles.show. 

export default Stylesheet.create({ stuff })
or 
const styles = Stylesheet.create({ stuff })
export default styles

Reply:
Agreed